lib/analytics/ga4/google_analytics_service.rb

```diff
@@ -0,0 +1,59 @@
+require "google/analytics/data/v1beta"
+require_relative "./request"
+require_relative "./page_data"
+require_relative "./client"
+
+require "dotenv"
+
+Dotenv.load
+
+module Analytics
+  module Ga4
+    class GoogleAnalyticsService
+      attr_accessor :ga_client, :all_data
+
+      OFFSET = 0
+      LIMIT = 10_000
+      OFFSET_INCREMENT = 10_000
+
+      def initialize
+        @ga_client = Analytics::Ga4::Client.new
```

Comment:
```suggestion
        @ga_client = Client.new
```

given we're already in this namespace.